src/sentry/search/events/datasets/discover.py

```diff
@@ -1617,8 +1618,13 @@ def _issue_filter_converter(self, search_filter: SearchFilter) -> Optional[Where
             except Exception:
                 raise InvalidSearchQuery(f"Invalid value '{group_short_ids}' for 'issue:' filter")
             else:
+                org = None
+                if groups:
+                    org = groups[0].project.organization
                 for group in groups:
-                    if group.issue_category == GroupCategory.PERFORMANCE:
+                    if group.issue_category == GroupCategory.PERFORMANCE and not features.has(
+                        "organizations:issue-platform-search-perf-issues", org
+                    ):
```

Comment:
did you push a change where filtering by `issue.id` would correctly query search_issues dataset and not errors? somehow just passing `issue.id` filter to discover for perf issues locally now works 😅

Reply:
Not intentionally haha, the closest thing I can think would be https://github.com/getsentry/sentry/pull/47406, but that was a couple weeks ago.